extensions/wikia/CategoryPagination/CategoryPaginationHooks.class.php

```diff
@@ -12,6 +12,11 @@ class CategoryPaginationHooks {
 	public static function onArticleFromTitle( &$title, &$article ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of returns of this function 4, down to the maximum allowed 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1142)